pandas/tests/io/excel/test_odf.py

```diff
@@ -33,7 +33,7 @@ def test_read_writer_table():
         columns=["Column 1", "Unnamed: 2", "Column 3"],
     )
 
-    result = pd.read_excel("writertable.odt", "Table1", index_col=0)
+    result = pd.read_excel("writertable.odt", sheet_name="Table1", index_col=0)
```

Comment:
Is `sheet_name` required to be passed like this or still OK positionally? I assume the latter from `allowed_args` being 2 but maybe am misreading

+/- 0 on making this required as a keyword argument. It would seem a rather natural positional argument so maybe not worth the churn, but not a strong opinion

Reply:
> Is sheet_name required to be passed like this or still OK positionally?

This is just a cleanup, I think it's more readable with keyword arguments.

I think exel files are very well known, so people will easily infer that the second parameter is the sheet name, and making it fail, e.g. in jupyter notebook could maybe be seen as too restrictive. This is not a strong opinion, so it making it a required kwarg is seen as better, I can do that.